.github/workflows/backend-integration-test.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         architecture: [linux/amd64]
         java-version: [ 8, 11, 17 ]
         test-mode: [ embedded, deploy ]
-        backend: [ jdbcBackend, kvBackend]
+        backend: [ mysql, h2]
```

Comment:
I think maybe we can reduce the pipeline to combine embedded with h2, and deploy with msyql, WDYT?

Reply:
In general, `test-mode` and `backend` are different things, I would rather to remove `embedded` mode in the `test-mode` to reduce pipelines as backend `h2` and `MySQL` will be run by the user in the real environment,  however embedded mode will never exists in real environment.